dojo/__init__.py

```diff
@@ -5,7 +5,6 @@
 from .celery import app as celery_app  # noqa
 
 default_app_config = 'dojo.apps.DojoAppConfig'
-
 __version__ = '2.0.0-dev'
```

Comment:
This delete is not needed

Reply:
I don't think this was from me actually. Prob due to some weird branching I did but I can fix that

Reply:
done